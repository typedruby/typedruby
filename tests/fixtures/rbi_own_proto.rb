def typed_in_stub(x)
  x + 1
end
