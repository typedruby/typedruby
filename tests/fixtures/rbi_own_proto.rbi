def typed_in_stub(String x) => String; end
