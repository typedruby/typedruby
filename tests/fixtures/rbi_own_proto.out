error: Could not match types:

        @ (builtin stdlib):722
    722 |    def +(String other) => String; end
                   ^^^^^^ String, with:
        @ __ROOT__/tests/fixtures/rbi_own_proto.rb:4
      4 |    x + 1
                 ^ Integer
        @ __ROOT__/tests/fixtures/rbi_own_proto.rb:4
      4 |    x + 1
             ^^^^^ in this expression
